content/english/arduino/Circuit-Basics.md

```diff
@@ -17,16 +18,17 @@ Imagine our circuit is a play. Before the final performance in front of an audie
 + The same is true on the red and blue lines on the left. However, the red and blue lines on the right and left are not connected to each other.
 + Components that are plugged into the same common connection are connected in the circuit
 
-![Alt Text: Picture of a breadboad](../img/breadboard.png)
+![Cannot load image](../img/img16.png)
```

Comment:
I think you want to undo the changes on this page.

Reply:
removed img17,18,19, and added img16